doc/lua_api.txt

```diff
@@ -3663,11 +3663,7 @@ Utilities
       {
           address = "127.0.0.1",     -- IP address of client
           ip_version = 4,            -- IPv4 / IPv6
-          min_rtt = 0.01,            -- minimum round trip time
```

Comment:
Uhm, won't that break backwards compatibility?
I'm using that, for example:
https://github.com/thomasrudin-mt/monitoring/blob/master/builtin/player_rtt.lua#L30

Reply:
Yes, you're right, it might break server mods.
Should I just let it return `avg_rtt - avg_jitter`? `min_rtt`/`max_rtt` will always have the value of the most extreme RTTs measured since joining, which is neither precise nor helpful.